Modules/Test/classes/class.ilTestEvaluationData.php

```diff
@@ -308,7 +308,7 @@ function addParticipant($active_id, $participant)
 	 * @param integer $active_id
 	 * @return ilTestEvaluationUserData
 	 */
-	function &getParticipant($active_id)
```

Comment:
This led to a weird behaviour when requesting particpant data (by active_id) for non existing particpants in consuming code, when the result of this method was assigned to a variable. In this case, a NULL array element was created automaically (in the instance) which results in fatal errors in different 'statistics' views.